clubs/tests/test_views.py

```diff
@@ -21,7 +21,8 @@ def setUp(self):
             location='Somewhere',
             latitude=5,
             longitude=6,
-            status='approved'
+            status='approved',
+            denial='approved'
```

Comment:
if you have a recommendation for how to deal with a conditional field (i.e. "only exists if other field has value X") I'm all ears. Eyes. Whatever works